ProductUtilities.cs

```diff
@@ -11,12 +11,12 @@ namespace Price_Calculator
     public  static  class ProductUtilities
     {
         private static Product product = new Product();
```

Comment:
why do you need to add the static instance? and why do you need the CreateInstance?